docs/Configuration.md

```diff
@@ -146,7 +176,41 @@ to specifying a profile JSON file containing the entire
 
 ## ZONEMASTER section
 
-TBD
+The ZONEMASTER section has several keys :
+`max_zonemaster_execution_time`,
+`number_of_professes_for_frontend_testing`,
+`number_of_professes_for_batch_testing`,
+`force_hash_id_use_in_API_starting_from_id`, `lock_on_queue`,
+`maximal_number_of_retries`.
+
+### max_zonemaster_execution_time
+
+This key allows to define the interval in seconds to reload unfinished
+queries.
+This assume that after `max_zonemaster_execution_time`, if the test is
+not finished, it has fail. And it will be run again up to
+`maximal_number_of_retries`.
+
+### maximal_number_of_retries
+
+Specifies the number of time a test is allowed to be run again if
+unfinished after `max_zonemaster_execution_time`.
+
+### number_of_professes_for_frontend_testing
+
+not used in the code
```

Comment:
Suggest:

Removed from all config files (normal and for Travis) and Config.pm. I do not think that will break anything.

Same thing for "number_of_processes_for_frontend_testing",